app.js

```diff
@@ -19,11 +19,19 @@
 
 const fs = require("fs");
 const { jsonToSchema } = require("./lib");
+const yargs = require("yargs/yargs");
+const { hideBin } = require("yargs/helpers");
+const argv = yargs(hideBin(process.argv)).argv;
```

Comment:
`yargs` has defaults which can help here IMHO.
```suggestion
const argv = yargs(hideBin(process.argv)).options({
  baseType: {
    description: "Name of the base schema type",
    required: false,
    alias: "b",
    default: ""
  },
  prefix: {
    description: "Prefix to add to every generated type",
    required: false,
    alias: "p",
    default: ""
  }
}).argv;
```